packages/react-front-kit/src/Components/CollapseButton/CollapseButtonControlled.tsx

```diff
@@ -68,6 +70,7 @@ export interface ICollapseButtonControlledProps<
   T extends number | string,
   C extends ElementType,
 > extends ICollapseButtonProps<T, C> {
+  collapseProps?: Omit<CollapseProps, 'in'>;
```

Comment:
Je pense que le bon typage serait plutôt:
```suggestion
  collapseProps?: Partial<CollapseProps>;
```